packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -120,7 +115,7 @@ function RevisionsButtons( {
 		<ol
 			className="edit-site-global-styles-screen-revisions__revisions-list"
 			aria-label={ __( 'Global styles revisions list' ) }
-			role="group"
```

Comment:
Should we use Composite component here and also improve the revisions list keyboard navigation?

Reply:
Thanks for the tip, I'll try out the Composite component. 


> Should we use Composite component here and also improve the revisions list keyboard navigation?

Are those two points mutually exclusive or is the latter a consequence of the former?

I'd like to keep this PR within the scope of https://github.com/WordPress/gutenberg/issues/66642.

Reply:
It can be iterated later, but it would probably change this added code too. It's fine to land this one first.